### PR TITLE
generalize tests

### PR DESCRIPTION
**Pull request by adamjanicki2**
Makes tests more general

### CODE REVIEW

File: src/childPaths.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ export default function childPaths (node) {
 		return [];
 	}
 
-	const childProperties = context.getProperties(node);
+	const childProperties = context.getChildProperties(node);
```
Comment (adamjanicki2) [inline src/childPaths.js:20]: This was a bug

File: src/children.js [hunk 2]
```diff
@@ -17,7 +17,7 @@ export default function children (node) {
 		return [];
 	}
 
-	const childProperties = context.getProperties(node);
+	const childProperties = context.getChildProperties(node);
```
Comment (adamjanicki2) [inline src/children.js:20]: Same bug

File: src/context.js [hunk 3]
```diff
@@ -17,7 +17,7 @@ export const defaults = {
 	 */
 	isNode: node => node && typeof node === "object"
 		&& Object.prototype.toString.call(node) === "[object Object]"
-		&& Object.getPrototypeOf(o).constructor?.name === "Object",
+		&& Object.getPrototypeOf(node).constructor?.name === "Object",
```
Comment (adamjanicki2) [inline src/context.js:20]: Also a bug

File: src/parents.js [hunk 4]
```diff
@@ -1,4 +1,4 @@
-import { updateParents } from "./updateParents.js";
+import updateParents from "./updateParents.js";
```
Comment (adamjanicki2) [inline src/parents.js:1]: Also a naming bug

File: test/utils/copy.js [hunk 5]
```diff
@@ -0,0 +1,3 @@
+export default function copy (tree) {
```
Comment (adamjanicki2) [inline test/utils/copy.js:1]: simple helper to deep copy a tree so fresh trees can be used during tests

**Comment by adamjanicki2** [comment]
@LeaVerou I'm not sure why the netlify build is failing?

**Comment by LeaVerou** [comment]
> @LeaVerou I'm not sure why the netlify build is failing?

Have you looked at  https://app.netlify.com/sites/treecle/deploys/65e6461124bd2a00090fdba6 ?

File: src/childPaths.js (hunk 1 above)
Comment (LeaVerou) [reply src/childPaths.js:20]: These kinds of things don't need to be part of PRs, you can commit them directly

File: src/index.js [hunk 6]
```diff
@@ -1,3 +1,3 @@
 export * from "./index-fn.js";
 import Treecle from "./Treecle.js";
-export default Treecle;
\ No newline at end of file
+export default Treecle;
```
Comment (LeaVerou) [inline src/index.js:3]: Same for this. You don't need to send a PR for adding a line break at the end of a fole!

**Review by LeaVerou — APPROVED** [review]

**Pull request merged**